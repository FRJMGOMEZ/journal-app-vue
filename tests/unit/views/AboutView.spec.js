
import AboutView from '@/views/AboutView.vue';
import { shallowMount } from '@vue/test-utils';

describe('AboutView component', () => {

    test('debe de hacer match con snapshot', () => {
        const wrapper = shallowMount(AboutView);
        expect(wrapper.html()).toMatchSnapshot();
    })
})
