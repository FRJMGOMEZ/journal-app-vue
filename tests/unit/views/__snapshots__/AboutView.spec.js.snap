// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutView component debe de hacer match con snapshot 1`] = `
<div class="about">
  <h1>This is an about page - </h1>
</div>
`;
